src/analytics.js

```diff
@@ -56,3 +57,28 @@ export function triggerAnalyticsEvent(nodeOrDoc, eventType, opt_vars) {
     analytics.triggerEvent(eventType, opt_vars);
   });
 }
+
+/**
+ * Helper method to create analytics element for specific root.
+ * @param {BaseElement} baseInstance
+ * @param {!JSONType} config
+ */
+export function createAnalyticsForExtension(baseInstance, config) {
```

Comment:
changed to `insertAnalyticsElement(element, config, parentDoc)` I don't see the need of type.